<?xml version="1.0"?>

<launch>
    <param name="H_min" value="153"/>
    <param name="H_max" value="179"/>
    <param name="S_min" value="152"/>
    <param name="S_max" value="255"/>
    <param name="V_min" value="205"/>
    <param name="V_max" value="255"/>


    <node name="platform_detection_real" pkg="vision" type="platform_detection_real" output="screen">
        <!--TARGET SIZE-->
        <param name="target_size" value="0.25"/> <!--Value in m2-->
        <param name="altura_platform" value="0.8"/> <!--Value in m-->

        <param name="min_num_sides" value="3"/>
        <param name="max_num_sides" value="4"/>

        <param name="iterations" value="3"/>
         
        <!--TOPICS-->
        <param name="platform_position_topic" value="/platform/estimated_position"/>
        <param name="image_topic" value="/ardrone/bottom/image_raw"/>
        <param name="cam_info_topic" value="/ardrone/bottom/camera_info"/>
        <param name="ardrone_navdata" value="/ardrone/navdata"/>        
        <param name="imu_topic" value="/ardrone/raw_imu"/>
    </node>

    

</launch>
